recipes/cuda-histogram/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cuda_histogram" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3672c474dd094e2fefd364757d8fb6aa016af6e2ffee356e2a9a78c88f58690
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win or osx or py<38]
+
+requirements:
+  host:
+    - python
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python
+    - awkward >=2.6.3
+    - cupy >=13.1.0
+    - cuda-nvcc
```

Comment:
Does the user need the CUDA compiler at runtime?

Reply:
I assume strictly no, but I'll let @Saransh-cpp comment on that.